tests/pre_merge/models/test_model_premerge.py

```diff
@@ -0,0 +1,57 @@
+"""Quick sanity check on models."""
```

Comment:
would test_model.py be enough since the parent directory already specifies the type of the test?

Reply:
I get pytest discovery issues if I have keep the names of both the files as `test_model.py`. Hence, I have given both of them a unique (but redundant) name.